purchase_landed_cost/wizard/import_landed_cost_pickings_wizard.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3
+
+from odoo import api, fields, models
+
+
+class ImportLandedCostPickingsWizard(models.TransientModel):
+    _name = 'import.landed.cost.pickings.wizard'
+
+    possible_picking_ids = fields.Many2many(
+        comodel_name="stock.picking", string="Possible pickings")
+    picking_ids = fields.Many2many(
+        comodel_name="stock.picking", string="Pickings",
+        domain="[('id', 'in', possible_picking_ids[0][2])]")
```

Comment:
I tested and apparently this does not work anymore.
I think we don't need `possible_picking_ids` and we can write a function that return directly the domain like this:
![image](https://user-images.githubusercontent.com/31387997/53350865-9fa4f400-3952-11e9-9db3-a008023b3a10.png)